pkg/cli/rebuild.go

```diff
@@ -64,10 +78,54 @@ func NewCmdRebuild() *cobra.Command {
 	cmd.Flags().StringVar(&prefix, "prefix", "", "Search prefix")
 	cmd.Flags().StringVar(&dir, "dir", "", "Output directory")
 	cmd.Flags().StringVar(&ignores, "ignores", "", "Comma separated list of ignore paths")
-	cmd.Flags().BoolVar(&summarize, "summarize", true, "Should include the summary file (required for future updates)")
+	cmd.Flags().BoolVar(&logUpdates, "log-updates", false, "Log all updated files to the console")
+	cmd.Flags().StringSliceVar(&checkGlobs, "check-glob", []string{}, "Report if any files matching the given globs were changed by this rebuild operation")
 	to = cmd.Flags().Int64("to", -1, "To version ID (optional)")
 
 	_ = cmd.MarkFlagRequired("project")
 
 	return cmd
 }
+
+func LogUpdates(diff *fsdiff_pb.Diff) {
+	for _, update := range diff.Updates {
+		fmt.Printf("%v %v\n", update.Action, update.Path)
+	}
+}
+
+func anyMatchedGlobs(globs []string, diff *fsdiff_pb.Diff) (bool, error) {
+	for _, glob := range globs {
+		for _, update := range diff.Updates {
+			match, err := doublestar.Match(glob, update.Path)
+			if err != nil {
+				return false, err
+			}
+			if match {
+				return true, nil
+			}
+		}
+	}
+
+	return false, nil
+}
+
+func LogIfGlobsMatched(globs []string, diff *fsdiff_pb.Diff) error {
+	if len(globs) > 0 {
+		matched, err := anyMatchedGlobs(globs, diff)
+		if err != nil {
+			return err
+		}
+		if matched {
+			fmt.Printf("GLOB-MATCH: files matching globs were matched during rebuild\n")
+		}
+	}
+	return nil
+}
```

Comment:
Aren't we trying to figure out if **only** specific files changed? This only tells us if specific files changed, not if they only changed. Unless I'm thinking about this wrong?